k4FWCore/components/IOSvc.cpp

```diff
@@ -42,6 +42,9 @@ StatusCode IOSvc::initialize() {
   if (!m_importedFromk4FWCore) {
     error() << "Use 'from k4FWCore import IOSvc' instead of 'from Configurables import IOSvc' to access the service"
             << endmsg;
+  }
+  if (m_outputType != "default" && m_outputType != "ROOT" && m_outputType != "RNTuple") {
+    error() << "Unknown input type: " << m_outputType << ", expected ROOT, RNTuple or default" << endmsg;
```

Comment:
`podio::Writer` allows also SIO. Is this intentional to exclude it?

Reply:
The demand for writing EDM4hep files using SIO is quite low. If someone wants it we can add it.

Reply:
Could be nice for some testing at some point, but I agree we can easily toggle this once it becomes necessary.